Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -0,0 +1,317 @@
+                <sDEF>
+                    <ROOT>
+                        <TCEforms>
+                            <sheetTitle>LLL:EXT:extkey/Resources/Private/Language/Backend.xlf:settings.registration.title</sheetTitle>
+                        </TCEforms>
+                        <type>array</type>
+                        <el>
+                            <!-- Add settings here ... -->
+
+                            <!-- Example setting: input field with name settings.timeRestriction -->
+                            <settings.timeRestriction>
+                                <TCEforms>
+                                    <label>LLL:EXT:example/Resources/Private/Language/Backend.xlf:settings.registration.timeRestriction</label>
+                                    <config>
+                                        <default></default>
+                                        <type>input</type>
+                                        <size>15</size>
+                                    </config>
+                                </TCEforms>
+                            </settings.timeRestriction>
+
+                            <!-- end of settings -->
+
+                        </el>
+                    </ROOT>
+                </sDEF>
+            </sheets>
+        </T3DataStructure>
+
+
+#. The configuration schema is attached to one or more plugins
+
+   The vendor name is **Myvendor**, the extension key is **example**
+   and the plugin name is **Registration**.
+
+   In :file:`Configuration/TCA/Overrides/tt_content.php` add the following:
+
+   .. code-block:: php
+
+       $GLOBALS['TCA']['tt_content']['types']['list']['subtypes_addlist']['example_registration'] = 'pi_flexform';
+       \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPiFlexFormValue(
+           // plugin signature: <extension key> '_' <plugin name in lowercase>
+           'example_registration',
+           // Flexform configuration schema file
+           'FILE:EXT:example/Configuration/FlexForms/Registration.xml'
+       );
+
+   .. tip::
+
+      The plugin signature is used in the database field `tt_content.list_type`
+      as well, when the tt_content record is saved. If you are confused about
+      how to handle underscores and upper / lowercase, check there to see
+      what your plugin signature is.
+
+      Also look on the page :ref:`extension-naming`.
+
+#. The settings can be read using :php:`$this->settings` in an
+   Extbase controller.
+
+   .. code-block:: php
+
+      $includeCategories = (bool) ($this->settings['includeCategories'] ?? false);
+
+
+.. todo: Add information about how to read settings apart from Extbase Controllers.
+
+
+More Schema Examples
+====================
+
+The definition of the data types and parameters used complies in large
+parts to the :ref:`column types defined by TCA <t3tca:columns-types>`.
+
+The settings must be added within the <el> element in the FlexForm configuration
+schema file.
+
+.. important::
```

Comment:
Should we move this note to `The settings can be read using $this->settings in an Extbase controller.`, in order to keep context?